meta-aos-rcar-gen4-dom0/recipes-core/runx/runx/0001-files-create-Update-create-script-to-start-unikernel.patch

```diff
@@ -0,0 +1,99 @@
+From 42335786b59ae588453197b92c840175907d643e Mon Sep 17 00:00:00 2001
```

Comment:
Usually patches are places under `files` folder.

Reply:
As I know, both approaches are widely used. Both files and ${PV} folders are scanned - so there is no difference. The advantage of ${PV} is that in case if you have more than 1 recipe in folder, which requires pathing, you can put them separately. please take a look into meta-vistualization/recipes-extended/nagios as an example.

Reply:
I know it. Do you plan to add another recipe to this folder? In most cases `files` are used. I prefer to use the common pattern.